__tests__/app.test.js

```diff
@@ -50,4 +50,45 @@ describe('app', () => {
         });
     });
 
+    
+    describe('/api/articles/:article_id', () => {
+        test('200 : responds with an article object with all the given properties', () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                const properties = ['article_id', 'title', 'topic', 'author', 'body', 'created_at', 'votes', 'article_img_url']
+                properties.forEach((property) => {
+                    expect(response.body).toHaveProperty(property);
+                });
+                
+            });
+        });
+        test('200 : responds with the correct article object', () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then((response) => {
+                expect(response.body.title).toEqual('Living in the shadow of a great man');
+                expect(response.body.article_id).toEqual(1);
+                expect(response.body.topic).toEqual('mitch');
+                expect(response.body.author).toEqual('butter_bridge');
+                expect(response.body.body).toEqual('I find this existence challenging');
+                expect(response.body.created_at).toEqual('2020-07-09T20:11:00.000Z');
+                expect(response.body.votes).toEqual(100);
+                expect(response.body.article_img_url).toEqual('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700');
+            });
```

Comment:
This is fine but look into something like expect.objectContaining or .toMatchObject which would allow for better testing later on.